### PR TITLE
feat: Add shellcheck as a step of GitHub action

### PR DESCRIPTION
**Pull request by jirihnidek**
* Added two new steps of "lint-code" job
  * Try to install shellcheck CLI tool during CI
  * Try to check rhc.bash using this tool during CI

### CODE REVIEW

**Comment by jirihnidek** [comment]
This is blocked by #50

**Review by ahitacat — APPROVED** [review]
LGTM, but can you fix the commits before merge it?

**Pull request merged**
